string command
uint16 parameter